uber/templates/base.html

```diff
@@ -53,7 +53,7 @@
                 {People: [
                     {Attendees: '../registration/'},
                     {Groups: '../groups/'},
-                    {'All Untaken Shifts': ''},
+                    {'All Untaken Shifts': '../jobs/everywhere'},
```

Comment:
Is there... any particular reason for this to only show up while at the con? Brent seemed to want it to work now.